src/resolvers/exotics/bitbucket-resolver.js

```diff
@@ -7,6 +7,20 @@ export default class BitbucketResolver extends HostedGitResolver {
   static hostname = 'bitbucket.org';
   static protocol = 'bitbucket';
 
+  static isVersion(pattern: string): boolean {
+    // bitbucket proto
+    if (pattern.startsWith('bitbucket:')) {
+      return true;
+    }
+
+    // bitbucket shorthand
+    if (/^[^:@%/\s.-][^:@%/\s]*[/][^:@\s/%]+(?:#.*)?$/.test(pattern)) {
```

Comment:
Can you move this outside of the function, make it ` const` and add some comments about this regular expression? It is not apparent what it is doing for someone not familiar with bitbucket.

Reply:
I did not understand that it was the shortest shortcut, the one that should only work with github.